[Doc] Replace gated models with Qwen in quickstart examples

cdc542559455 opened the pull request:
## Summary

This PR updates quickstart documentation examples to use open Qwen models instead of gated Meta-Llama models, removing the HF_TOKEN dependency and improving the getting started experience.

## Changes

- **share_kv_cache.rst**: Replace `meta-llama/Meta-Llama-3.1-8B-Instruct` with `Qwen/Qwen2.5-7B-Instruct`
- **disaggregated_prefill.rst**: Update to use `Qwen/Qwen2.5-7B-Instruct` and add:
  - Version information (tested with vLLM 0.11.0, LMCache 0.3.9)
  - Fixed port numbers for consistency
  - Added `nixl_backends` configuration
- Add `LMCACHE_USE_EXPERIMENTAL=True` environment variable to all examples
- Remove HF_TOKEN requirements from examples

## Testing

- Tested with vLLM 0.11.0 and LMCache 0.3.9
- Verified all examples work with Qwen/Qwen2.5-7B-Instruct model
- Confirmed no HF_TOKEN authentication needed

## Motivation

The current quickstart examples use gated models (Meta-Llama-3.1-8B-Instruct) that require users to authenticate with HuggingFace tokens, creating friction for new users. Using open Qwen models makes the quickstart more accessible and user-friendly.

Commit:
[Doc] Replace gated models with Qwen in quickstart examples

- Update share_kv_cache.rst to use Qwen/Qwen2.5-7B-Instruct
- Update disaggregated_prefill.rst to use Qwen/Qwen2.5-7B-Instruct
- Add LMCACHE_USE_EXPERIMENTAL=True environment variable
- Add version information (tested with vLLM 0.11.0, LMCache 0.3.9)
- Fix port numbers and add nixl_backends configuration in disagg_prefill
- Remove HF_TOKEN dependency from examples

Signed-off-by: Loki Chen <lokichen@Mac.attlocal.net>

kobe0938 commented on docs/source/getting_started/quickstart/disaggregated_prefill.rst:
```diff
@@ -150,26 +151,26 @@ Step-by-Step Setup
     cd vllm/benchmarks
-    python benchmark_serving.py --port 9000 --seed $(date +%s) \
-        --model meta-llama/Llama-3.1-8B-Instruct \
+    python benchmark_serving.py --port 9100 --seed $(date +%s) \
```
`benchmark_serving.py` is deprecated, please update

cdc542559455 replied:
Removed the benchmark.

kobe0938 reviewed (changes requested):
In general looks good. Could you rerun other examples like https://docs.lmcache.ai/getting_started/quickstart/multimodality.html and https://docs.lmcache.ai/getting_started/quickstart/offload_kv_cache.html to check if anything is outdated, depredated, or improper?

Commit:
[Doc] Update multimodality example to Qwen2-VL-2B and remove deprecated benchmark_serving.py

- Replace audio model (ultravox) with vision-language model (Qwen/Qwen2-VL-2B-Instruct)
- Add version info note (vLLM 0.11.0, LMCache 0.3.9)
- Simplify multimodal example with text-based test
- Update sample output with generic LMCache monitoring logs
- Remove deprecated benchmark_serving.py reference from disaggregated prefill docs

Signed-off-by: Loki Chen <cdc542559455@gmail.com>
Signed-off-by: Loki Chen <lokichen@Mac.attlocal.net>

Commit:
[Doc] Add example output to multimodality quickstart

- Add JSON response example showing actual generated text
- Update LMCache monitoring logs with complete sequence
- Show cache hit on second request
- Based on tested Qwen2-VL-2B-Instruct model output

Signed-off-by: Loki Chen <cdc542559455@gmail.com>
Signed-off-by: Loki Chen <lokichen@Mac.attlocal.net>

cdc542559455 commented:
> In general looks good. Could you rerun other examples like https://docs.lmcache.ai/getting_started/quickstart/multimodality.html and https://docs.lmcache.ai/getting_started/quickstart/offload_kv_cache.html to check if anything is outdated, depredated, or improper?

https://docs.lmcache.ai/getting_started/quickstart/offload_kv_cache.html already adopted Qwen 3. https://docs.lmcache.ai/getting_started/quickstart/multimodality.html is also fine. I have specified `Qwen/Qwen2-VL-2B-Instruct`

kobe0938 commented:
> https://docs.lmcache.ai/getting_started/quickstart/offload_kv_cache.html already adopted Qwen 3. https://docs.lmcache.ai/getting_started/quickstart/multimodality.html is also fine. I have specified `Qwen/Qwen2-VL-2B-Instruct`

Well, `Qwen/Qwen2-VL-2B-Instruct` is specified in multimodality doc however there's no image/audio being sent to verify the capability. The previous example is deleted. Could you add them back?
The vllm `benchmark_serving.py` was replaced by `vllm bench serve`, could you add it?
